README.md

```diff
@@ -737,3 +738,131 @@ git add <the files>
 git commit
 git push origin develop
 ```
+
+## Updating a production instance
+
+Every now and then, the production instance needs to be updated, so the server
+can get the latest security patches, and the Research Software Directory
+software itself can be updated to include the latest features.
+
+The steps below differentiate between the old and the new instance of the Research
+Software Directory; the old instance has IP ``35.156.38.208``, the new one has
+IP ``3.122.233.225``. Your IP addresses will likely be different.
+
+1. Make a new Amazon instance by following the notes above. Some things to think about:
+    - Reuse the existing security group.
+    - Reuse the existing key pair.
+    - Verify that you're allowed to ssh into the new instance.
+1. Download the ``rsd-secrets.env`` file from the old instance.
+
+    ```
+    # something like:
+    $ cd $(mktemp -d)
+
+    $ scp -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem \
+      ubuntu@35.156.38.208:/home/ubuntu/rsd/rsd-secrets.env .
+    ```
+1. Download files related to SSL certificates from the old instance.
+
+    ```
+    $ scp -r -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem \
+      ubuntu@35.156.38.208:/home/ubuntu/rsd/docker-volumes/cert .
+    ```
+
+1. Upload ``rsd-secrets.env`` to the new Research Software Directory
+   instance.
+
+    ```
+    $ scp -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem rsd-secrets.env \
+    ubuntu@3.122.233.225:/home/ubuntu/rsd/
+    ```
+
+1. Upload the files related to SSL certificates to the new Research Software Directory
+   instance.
+
+    ```
+    # remove the cert directory from /home/ubuntu/rsd/docker-volumes/ if it
+    # exists on the new machine
+
+    $ scp -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem cert \
+    ubuntu@3.122.233.225:/home/ubuntu/rsd/docker-volumes/
+    ```
+
+1. Stop new additions to the database in the old research software
+   directory instance by stopping the ``rsd-admin`` service.
+
+    ```
+    $ ssh -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem ubuntu@35.156.38.208
+    $ cd rsd
+    $ docker-compose stop rsd-admin
+    ```
+
+1. Download the data from the old Research Software Directory instance by
+   manually triggering a backup.
+
+    ```
+    # Add the environment variables to the shell:
+    $ source rsd-secrets.env
+
+    # Start the backup itself:
+    $ docker-compose exec backup /bin/sh /app/backup.sh
+    ```
+
+1. Manually download the data from S3 to your local system, and extract
```

Comment:
I do not like the S3 copy actions, they feel under documented and specific to the NLeSC installation.

Can we instead get the backup tarball from the back up container, using the following commands:

On old instance:
```sh
# Copy backup tarball out of Docker image
docker cp $(docker-compose ps -q backup):/app/rsd-backup.tar.gz /home/ubuntu/rsd/
```

On laptop:
```sh
# Transfer backup tarball from old to new instance
scp -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem cert \
    ubuntu@35.156.38.208:/home/ubuntu/rsd/rsd-backup.tar.gz \
    ubuntu@3.122.233.225:/home/ubuntu/rsd/sd-backup.tar.gz
```
On new instance:
```sh
# Unpack backup tarball
tar -xvzf rsd-backup.tar.gz -C database/db-init/
```